Dockerfile

```diff
@@ -0,0 +1,164 @@
+FROM ubuntu:xenial
+MAINTAINER Susanna Kiwala <ssiebert@wustl.edu>
+
+LABEL \
+    description="Image for tools used in the CLE"
+
+RUN apt-get update -y
```

Comment:
Yeah, I wasn't sure about that either. It's kinda nice to have them separate to know which software has which dependencies. But it's certainly more efficient to combine and deduplicate these.